WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayCompleteViewModel.swift

```diff
@@ -47,7 +44,13 @@ final class SetUpTapToPayCompleteViewModel: PaymentSettingsFlowPresentedViewMode
     /// Notifies the viewModel owner if a change occurs via didChangeShouldShow
     ///
     private func reevaluateShouldShow() {
-        let newShouldShow: CardReaderSettingsTriState = connectedReader
+        let newShouldShow: CardReaderSettingsTriState
+        switch (doneWasTapped, connectedReader) {
+        case (true, _):
+            newShouldShow = .isFalse
+        case (false, _):
+            newShouldShow = connectedReader
+        }
```

Comment:
`connectedReader` is not currently being used in the switch statement, I'm guessing we'll be using this to perform other logic moving forward?
```suggestion
        switch doneWasTapped {
        case true:
            newShouldShow = .isFalse
        case false:
            newShouldShow = connectedReader
        }
```

Reply:
I think that was my intention but it didn't end up being needed. Fixed in 885e69e